### PR TITLE
Get user info/get values from fields

### PR DESCRIPTION
**Pull request by edgardovbak**
Hi

Changes 
- Convert to typescript
- jwt authantication
- get user info

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,17 +6,49 @@
...
+    "postcss-loader": "2.0.8",
+    "promise": "8.0.1",
+    "raf": "3.4.0",
+    "react-dev-utils": "4.2.1",
     "react-dom": "^16.2.0",
     "react-redux": "^5.0.7",
-    "react-scripts-ts": "2.14.0",
     "react-timestamp": "^4.4.0",
-    "sn-client-js": "^3.0.3"
+    "sn-client-js": "^3.0.3",
```
Comment (gallayl) [inline package.json]: I think sn-client-js should be removed, use the scoped packages

File: src/components/EditProfil.tsx [hunk 2]
```diff
@@ -5,53 +5,72 @@ import { Link } 					from 'react-router-dom';
...
+
 		return (
 			<div className="profil">
 				<div className="user" >
 					<UserAvatar />
 					<div className="user__global_info">
 						<fieldset>
 							<legend>User name</legend>
-							<input id="userName" type="text" placeholder="User name" defaultValue={this.props.user.FullName} />
+							<input id="userName" type="text" ref={(input) => { this.FullName = input as any; }} placeholder="User name" defaultValue={this.props.user.FullName} />
```
Comment (gallayl) [inline src/components/EditProfil.tsx]: You can use HTMLInputElement type for better intellisense support in such cases

File: src/App.tsx [hunk 3]
```diff
@@ -17,16 +17,21 @@ import Header                               from './components/Header';
 import Profil                               from './components/Profil';
 import EditProfil                           from './components/EditProfil';
 import { Login }                            from './components/Login';
+import { PathHelper } from '@sensenet/client-utils';
 
 // save config 
 const DATA = require('./config.json');
```
Comment (gallayl) [inline src/App.tsx]: Environment based URLs can be used in long term with React + Webpack with a fallback to the development URL. An example: https://github.com/SenseNet/sn-dms-demo/blob/master/src/index.tsx#L28 and

**Review by gallayl — CHANGES_REQUESTED** [review]

**Review by edgardovbak — COMMENTED** [review]
- remove sn-client-js from package.json
- use htmlInputElement type in editprofile
- change repoUrl in App

**Pull request merged**